Expand_cards/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Expand's card</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="script.js" defer></script>
+  </head>
+  <body>
+    <div class="container">
+      <div
+        class="panel active"
+        style="
+          background-image: url(https://images6.alphacoders.com/656/thumb-1920-656029.png);
+        "
+      >
+        <h3>Anime x sliders</h3>
+      </div>
+
+      <div
+        class="panel"
+        style="
+          background-image: url(https://images.alphacoders.com/103/thumb-1920-1032111.png);
+        "
+      >
+        <h3>Kamado Tanjiro</h3>
```

Comment:
The heading you have added are not visible properaly because of the color of text, kindly create dark background for the white text, or any other so that the text will be visible properly.